### PR TITLE
Added group key names to private message sqs messages

### PR DESCRIPTION
**Pull request by summraznboi**
We need group key names for future-proofing from messaging to special group keys (e.g. base/default), versus legitimate group messaging in the future.

### CODE REVIEW

File: lib/sqs.go [hunk 1]
```diff
@@ -131,10 +134,17 @@ func (sqsQueue *SQSQueue) SendSQSTxnMessage(mempoolTxn *MempoolTx) {
 		glog.Errorf("SendSQSTxnMessage: Error marshaling transaction JSON : %v", err)
 	}
 
+	messageAttributes := make(map[string]types.MessageAttributeValue)
+	messageAttributes["messageId"] = types.MessageAttributeValue{
+		DataType:    aws.String("String"),
+		StringValue: aws.String(uuid.NewString()),
```
Comment (lgbello) [inline lib/sqs.go:140]: Seems unnecessary to have DataType: String when the field is called “StringValue”

**Review by lgbello — COMMENTED** [review]
The rest looks good

File: lib/sqs.go (hunk 1 above)
Comment (lgbello) [reply lib/sqs.go:140]: I would either remove datatype or just call the second field “value”

Comment (summraznboi) [reply lib/sqs.go:140]: This is actually needed, since DataType can also be Number, but would also use StringValue to specify. (This is aws sqs format).

**Review by cody85 — APPROVED** [review]

File: lib/sqs.go (hunk 1 above)
Comment (lgbello) [reply lib/sqs.go:140]: Ah ok. Looking at where this is used in transactions-worker this makes sense now. Approving

**Review by lgbello — APPROVED** [review]

**Pull request merged**